Make AWS SDK v1 and v2 instrumentation examples more consistent (#370)
aws/aws-xray-sdk-go: README.md

```go
func getExample(ctx context.Context) ([]byte, error) {
  resp, err := ctxhttp.Get(ctx, xray.Client(nil), "https://aws.amazon.com/")
  if err != nil {
    return nil, err
  }
  return ioutil.ReadAll(resp.Body)
}
```

**AWS SDK Instrumentation**

```go

> func main() {

Author: Agreed. To not expand this PR's scope, replacing all the double spaces can be left in a future update.

	// Create a segment
	ctx, root := xray.BeginSegment(context.TODO(), "AWSSDKV1_Dynamodb")
	defer root.Close(nil)

	sess := session.Must(session.NewSession())
	dynamo := dynamodb.New(sess)
	// Instrumenting with AWS SDK v1:

> 	// Wrap the client object with the xray.AWS()

Reviewer: ```suggestion
	// Wrap the client object with `xray.AWS()`
```

	xray.AWS(dynamo.Client)

> 	// Use the withContext version of the ListTables call method

Reviewer: ```suggestion
	// Use the `-WithContext` version of the `ListTables ` method
```

	output := dynamo.ListTablesWithContext(ctx, &dynamodb.ListTablesInput{})
	doSomething(output)
}
```

> *Segment creation is not necessary in AWS Lambda functions, where segments are created automatically*

Reviewer: ```suggestion
*Segment creation is not necessary in an AWS Lambda function, where the segment is created automatically*
```

	}
}
```

> *Segment creation is not necessary in AWS Lambda functions, where segments are created automatically*

Reviewer: ```suggestion
*Segment creation is not necessary in an AWS Lambda function, where the segment is created automatically*
```
